src/topology/continuous_function/algebra.lean

```diff
@@ -340,6 +364,14 @@ instance module : module R C(α, M) :=
 
 variables (R)
 
+/-- Composition on the left by a continuous linear map, as a `linear_map`.
+Similar to `linear_map.comp_left`. -/
+@[simps]
+def _root_.continuous_linear_map.comp_left (α : Type*) [topological_space α] (g : M →L[R] M₂) :
```

Comment:
```suggestion
def _root_.continuous_linear_map.comp_left_continuous (α : Type*) [topological_space α] (g : M →L[R] M₂) :
```
since I'd expect `comp_left` to refer to `(α → M) →ₗ[R] (α → M₂)` like it does for all the other defs in this PR